server/services/permissionsService.ts

```diff
@@ -123,9 +124,12 @@ export default class PermissionsService {
   }
 
   public getLocationPermissions(user: HmppsUser, prisoner: Prisoner): Permissions {
-    const accessCode = getOverviewAccessStatusCode(user, prisoner)
+    const accessCode = getOverviewAccessStatusCode(user, prisoner, { allowGlobal: false })
     if (accessCode !== HmppsStatusCode.OK) return { accessCode }
 
+    // Global is not allowed for locations
```

Comment:
I don't think this is needed if we change `getOverviewAccessStatusCode` as per other comment?